shared/src/packages/meditation/libs/validation-schemas/create-meditation.validation-schema.ts

```diff
@@ -0,0 +1,37 @@
+import joi from 'joi';
+
+import { ContentType } from '#libs/enums/enums.js';
+
+import { MeditationValidationMessage } from '../enums/enums.js';
+import { type MeditationCreateValidation } from '../types/types.js';
+
+const MAXIMUM_FILE_SIZE = 10_485_760;
```

Comment:
should be in MeditationEntryValidationRule

Reply:
Done